content/docs/latest/spire-helm-charts-hardened-about/exposing.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Exposing Services
+short: Exposing Services
+description: How to expose SPIFFE/SPIRE services outside of Kubernetes
+kind: spire-helm-charts-hardened-about
+weight: 150
+aliases:
+    - /docs/latest/helm-charts-hardened/exposing
+---
+
+# Default
+
+By default no services are exposed outside the Kubernetes cluster.
+
```

Comment:
```suggestion

By default no SPIRE services are exposed outside the Kubernetes cluster. The below sections cover how to expose them.

```

Reply:
This looks good but removes a link to show this being Default behavior. Should we leave that part in?

Reply:
ok with me